pkg/rpcserver/rpcserver.go

```diff
@@ -424,6 +428,13 @@ func (serv *Server) CreateInstance(name string, injectExec chan<- bool, updInfo
 		panic(fmt.Sprintf("duplicate instance %s", name))
 	}
 	serv.runners[name] = runner
+	id, ok := serv.ids[name]
```

Comment:
Yes, I think we should replace names with dense indices everywhere.
Names now only introduce additional entities and produce false impression that they are somehow different from indices.
Should I do this in this change?

Reply:
What do you mean by dense indices?

Reply:
`[0,VM count)`
(as opposed to giving VM new index after reboot)

Reply:
If we are limited to `[0;count)`, it makes it trickier to track whether an instance with the same index is the same one and should be avoided or it has already rebooted and we can safely execute the request there. Unique indexes would make life much simpler in this regard.

Reply:
Yes, but queue.Distributor needs dense IDs, while there is currently no use for unique IDs.
I've added commit that switches to IDs instead of names.
If we ever need unique IDs, I think we need to added them on top. But unique IDs are trivial to add, that's just a counter, and it does not need any help from executor/etc.